src/main/java/properties/EnvironmentConfig.java

```diff
@@ -0,0 +1,84 @@
+package properties;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Properties;
+
+/***
+ * Project environment data
+ */
+public class EnvironmentConfig {
+
+    private static final String MAIN_URL = "url";
+    private static final String REGISTRATION_URL = "regURl";
+    private static final String CATALOGUE_URL = "catalogueURL";
+    private static final String CART_URL = "cartURL";
+    private static final String PASSWORD = "password";
+    private static final String LOGIN = "login";
+
+    private final static String ENVIRONMENT_PROPERTIES_PATH = "src/main/resources/application-%s.properties";
+    private final static Properties properties;
+
+    @Getter
+    @AllArgsConstructor
+    enum Environment {
+        QA("qa"),
+        DEV("dev"),
+        LOCAL("local");
+
+        String name;
+
+        public static Environment setEnvironment() {
+            String defaultEnvironment = System.getenv("environment");
+            String environment = defaultEnvironment == null ? properties.getProperty("environment") : defaultEnvironment;
```

Comment:
You specify env in pom.xml, isn't that default one? how do you access that?

Reply:
Why do you access env name from env-bases file? at this point program doesn't even know what env is chosen...

Reply:
> Why do you access env name from env-based file? at this point program doesn't even know what env is chosen...

My bad, I guess now it is fixed.